api/nodes/views.py

```diff
@@ -361,3 +362,30 @@ def get_queryset(self):
                 files.append(self.get_file_item(waterbutler_data, cookie, obj_args))
 
         return files
+
+class NodeLogList(generics.ListAPIView, NodeMixin):
+    """ Recent Log Activity
+
+    This allows users to be able to get log information. This will allow more interesting
+    use cases for the API. Also this will be necessary if we want to be able to use the
+     v2 api for the project summary page.
+
+    """
+    serializer_class = NodeLogSerializer
+
+    log_lookup_url_kwarg = 'node_id'
+
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        ContributorOrPublic,
+    )
+
+    def get_queryset(self):
+        log_id = [self.get_node()._id]
+        query = Q('__backrefs.logged.node.logs', 'in', log_id)
```

Comment:
This seems like the long way around. Couldn't you just set log id to the id and query if the logs id equals log_id?